uber/models.py

```diff
@@ -942,7 +942,7 @@ def is_setup(self):
 
     @property
     def is_teardown(self):
```

Comment:
There are three hard problems in computer science:
- cache invalidation
- what to name your variables
- off-by-one errors